subprojects/ide/src/test/groovy/org/gradle/plugins/ide/internal/tooling/eclipse/EclipseModelBuilderTest.groovy

```diff
@@ -66,6 +67,20 @@ class EclipseModelBuilderTest extends AbstractProjectBuilderSpec {
         eclipseModel.projectNatures.collect { it.id } == ['nature.a', 'nature.b']
     }
 
+    def "can modify natures before and after merging"() {
```

Comment:
❌ Please add test coverage for all attributes that can be adjusted in in `project.file.beforeMerged` and `project.file.whenMerged`: 
- natures
- build commands
- linked resources